lib/email_api/emailer.py

```diff
@@ -114,9 +117,20 @@ def build_email_body(self, templates: List[EmailTemplateDetails], as_html):
                 msg_body += self._template_handler.render_plaintext_template(
                     template_details
                 )
-
         if as_html:
-            return MIMEText(msg_body, "html")
+            # wrap the email in our own styling
+            wrapper_template = EmailTemplateDetails(
+                template_name="wrapper", template_params={"body": msg_body}
+            )
+            html_body = self._template_handler.render_html_template(wrapper_template)
+
+            # convert style tags to inline styles for emails
+
+            # weird issue where pylint can't find the module - works fine though
+            # pylint:disable=no-member
+            inliner = css_inline.CSSInliner(keep_style_tags=True)
```

Comment:
If you change the import:
```suggestion
            inliner = CSSInliner(keep_style_tags=True)
```